server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const app = express();
+app.use(cors());
+
+const PORT = process.env.PORT;
+
+const weatherData = require('./data/weather.json');
+const { request } = require('express');
+
+
+app.get('/temp', (request, response) => { response.send('This is a placeholder string') });
+app.get('/weather', handleGetWeather);
+
+weatherDescription = () => {
```

Comment:
I know you are still workin so nice job so far on lab!

For line 18, I think you want to add a `const` here as well 👍

Reply:
Your `.env` was also pushed up to git so you will need to delete and then make sure it is properly added to gitignore. ( I commented here for that since I couldn't comment in file for .env) 👍